packages/gparted.rb

```diff
@@ -45,12 +34,14 @@ def self.build
   end
 
   def self.install
-    system "make DESTDIR=#{CREW_DEST_DIR} install"
-  end
-  
-  def self.postinstall
-    puts "To run gparted, execute the following:".orange
-    puts "xhost si:localuser:root".lightblue
-    puts "sudo gparted".lightblue
+    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
+    FileUtils.mv "#{CREW_DEST_PREFIX}/bin/gparted", "#{CREW_DEST_PREFIX}/bin/gparted.orig"
+    system "cat <<'EOF'> gparted
+#!/bin/bash
+DISPLAY=:0
```

Comment:
Do we need to set DISPLAY? What if one is connected via ssh?

Can we just use the sudo env variables by doing something like a ```sudo -E```

Reply:
Done, use `sudo -E`